sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/Acl.java

```diff
@@ -42,14 +42,34 @@
 
 
   /**
-   * The gRPC status code to respond with when access is denied.
+   * The status code to respond with when access is denied.
    *
-   * By default, this will be 7 (permission denied), but alternatives might include 16 (unauthenticated) or 5 (not
-   * found). If 0, indicates that the code should be inherited from the parent (regardless of the inherit field).
+   * By default, this will be 'Forbidden', but alternatives might include 'Authentication required' or 'Not
+   * Found'. If 'Inherited', indicates that the code should be inherited from the parent (regardless of the inherit field).
    *
-   * When HTTP transcoding is in use, this code will be translated to an equivalent HTTP status code.
    */
-  int denyCode() default 0;
+  DenyStatusCode denyCode() default  DenyStatusCode.FORBIDDEN_403;
+
+  enum DenyStatusCode {
+    INHERITED(0),
+    BAD_REQUEST_400(3),
+    FORBIDDEN_403(7),
+    NOT_FOUND_404(5),
+
+    AUTHENTICATION_REQUIRED_407(16),
+    CONFLICT_409(6),
+    INTERNAL_SERVER_ERROR_500(13),
+    SERVICE_UNAVAILABLE_503(14),
+    GATEWAY_TIMEOUT_504(4);
```

Comment:
yep, that's why. 
I thought about it removing it but I wanted to see what you guys think about it. I like to have the reference namely because is an indicator that it is an HTTP status code but I agree it doesn't look great.

Reply:
yeah, I mean.. it was certainly more explicit but also felt weird.. maybe we can have the input from someone else? since this is user facing could be good to have another pair of eyes maybe? @octonato WDYT?